### PR TITLE
Cknieling/add about page

### PR DESCRIPTION
**Pull request by chr11115**
- Add Help menu
- Add About page

### CODE REVIEW

File: src/serlio/scripts/serlioCreateUI.mel [hunk 1]
```diff
@@ -140,6 +140,71 @@ global proc createMaterialNode(string $materialType) {
 	select -r $select;
 }
 
+global proc createHelpMenuItems(){
+	string $serlioWebsite = "https://esri.github.io/cityengine/serlio";
+	string $cgaReference = "https://doc.arcgis.com/en/cityengine/latest/cga/cityengine-cga-introduction.htm";
+	string $rulePackageManual = "https://doc.arcgis.com/en/cityengine/latest/help/help-rule-package.htm";
+
+	menuItem -label "Serlio Website" -c ("launch -web \"" + $serlioWebsite + "\"");
```
Comment (mistafunk) [inline src/serlio/scripts/serlioCreateUI.mel:148]: I see this warning when opening a link (Maya 2022 Linux):
````
This tool has been deprecated, use 'gio open' instead.
See 'gio help open' for more info.
````

**Review by mistafunk — CHANGES_REQUESTED** [review]
See inline

**Comment by chr11115** [comment]
- Add missing legal text
- Fix icon on linux
- Add visual separator in serlio menu
- Rename "About" to "About Serlio"
- Add "?" Icon on Help menu
- Use xpm file in icons folder instead of png in docs folder

File: src/serlio/scripts/serlioCreateUI.mel (hunk 1 above)
Comment (mistafunk) [reply src/serlio/scripts/serlioCreateUI.mel]: warning still there, but we can ignore this for now.

**Pull request merged**